tools/individual-pkg-gen/bin/individual-pkg-gen.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Minor: rename to `individual-pkg-gen`. Drop the `.sh` suffix.

Also make sure the binary has execute bit set.

Reply:
Done.